migrations/2_deploy_dharma_contracts.js

```diff
@@ -1,22 +1,44 @@
+const OWNERS = require("./OWNERS");
+
 module.exports = (deployer, network, accounts) => {
+    // Import the Dharma contracts.
     const PermissionsLib = artifacts.require("PermissionsLib");
-    const DummyContract = artifacts.require("DummyContract");
     const DebtRegistry = artifacts.require("DebtRegistry");
     const DebtToken = artifacts.require("DebtToken");
     const DebtKernel = artifacts.require("DebtKernel");
     const RepaymentRouter = artifacts.require("RepaymentRouter");
     const TokenTransferProxy = artifacts.require("TokenTransferProxy");
     const MultiSigWallet = artifacts.require("MultiSigWallet");
 
+    // We switch on the network to ensure we're configuring our MultiSigWallet
+    // accordingly.
+    let owners;
+    switch (network) {
+        case "live":
+            owners = OWNERS.owners;
+            break;
+        case "kovan":
```

Comment:
Are we confident these are the names truffle uses to represent kovan / testrpc?

Reply:
Do you know otherwise? Link?

Reply:
We can also just special case the `live` network and assign the `accounts` for everything else

Reply:
I don't -- just wasn't totally sure how the `network` string gets assigned.

Also, this is a bit of nit, but I'd prefer if string literals were subbed in for constant -- i.e. `const LIVE_NETWORK_ID = "live";` etc.

Reply:
Yeah, i think special casing just `"live"` make sense.

Reply:
Not a nit at all, let's definitely make a constant.